<launch>

  <master auto="start"/>

  <node name="map_server" pkg="map_server" type="map_server" args="$(find waypoint_movement_test)/stage/map.yaml" />


  <include file="$(find waypoint_movement_test)/amcl.launch" />
  
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

    <rosparam file="$(find waypoint_movement_test)/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find waypoint_movement_test)/costmap_common_params.yaml" command="load" ns="local_costmap" />

    <rosparam file="$(find waypoint_movement_test)/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find waypoint_movement_test)/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find waypoint_movement_test)/base_local_planner_params.yaml" command="load" />
  
  </node>

</launch>
